pkg/client/status/v2.go

```diff
@@ -0,0 +1,99 @@
+package apistatus
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neofs-api-go/v2/status"
+)
+
+// StatusV2 defines a variety of Status instances compatible with NeoFS API V2 protocol.
+//
+// Note: it is not recommended to use this type directly, it is intended for documentation of the library functionality.
+type StatusV2 interface {
+	Status
+
+	// Returns the status as github.com/nspcc-dev/neofs-api-go/v2/status.Status message structure.
```

Comment:
```suggestion
	// ToStatusV2 returns the status as github.com/nspcc-dev/neofs-api-go/v2/status.Status message structure.
```

Reply:
This is not really necessary, but I will add.